Controller/ContentTypeController.php

```diff
@@ -130,7 +130,7 @@ public function createContentTypeAction($contentTypeGroupId, $languageCode = nul
             $contentTypeGroup = $this->contentTypeService->loadContentTypeGroup($contentTypeGroupId);
 
             $contentTypeCreateStruct = new ContentTypeCreateStruct([
-                'identifier' => 'new_content_type',
+                'identifier' => '__new__' . md5(microtime(true)),
```

Comment:
It's done this way because that's how @lolautruche did it for section creation. The string is not supposed to be visible in the UI, and the user is supposed to replace it with something more readable anyway (similar to the content type name). It's mainly there to enable quick creation of test types, I guess.